storage/aptosdb/src/state_store/mod.rs

```diff
@@ -141,7 +141,7 @@ impl StateStore {
             JellyfishMerkleTree::new(self).get_with_proof(state_key.hash(), version)?;
         Ok((
             match leaf_data {
-                Some(x) => Some(self.get_value_at_version(&x.1)?),
+                Some(x) => Some(self.get_existing_value_by_version(&x.1 .0, x.1 .1)?),
```

Comment:
too ugly.. give them names.

Reply:
they are tuple. how do you assign names to tuple?hhh